src/main/java/net/sf/jabref/gui/preftabs/ExternalTab.java

```diff
@@ -113,22 +172,66 @@ public void setValues() {
         openFoldersOfAttachedFiles.setSelected(prefs.getBoolean(JabRefPreferences.OPEN_FOLDERS_OF_ATTACHED_FILES));
 
         citeCommand.setText(prefs.get(JabRefPreferences.CITE_COMMAND));
+
+        defaultConsole.setSelected(Globals.prefs.getBoolean(JabRefPreferences.USE_DEFAULT_CONSOLE_APPLICATION));
+        specifiedConsole.setSelected(!Globals.prefs.getBoolean(JabRefPreferences.USE_DEFAULT_CONSOLE_APPLICATION));
+        consoleEmulatorPath.setText(Globals.prefs.get(JabRefPreferences.CONSOLE_APPLICATION));
+
+        updateEnableStatus();
     }
 
     @Override
     public void storeSettings() {
         prefs.put(JabRefPreferences.EMAIL_SUBJECT, emailSubject.getText());
         prefs.putBoolean(JabRefPreferences.OPEN_FOLDERS_OF_ATTACHED_FILES, openFoldersOfAttachedFiles.isSelected());
         prefs.put(JabRefPreferences.CITE_COMMAND, citeCommand.getText());
+        prefs.putBoolean(JabRefPreferences.USE_DEFAULT_CONSOLE_APPLICATION, defaultConsole.isSelected());
+        prefs.put(JabRefPreferences.CONSOLE_APPLICATION, consoleEmulatorPath.getText());
     }
 
     @Override
     public boolean validateSettings() {
+        if (!consoleEmulatorPath.getText().trim().isEmpty()) {
+            Path path = Paths.get(consoleEmulatorPath.getText());
+
+            if (!Files.exists(path) || Files.isDirectory(path) || !path.isAbsolute()) {
+                JOptionPane.showMessageDialog(null,
+                        Localization.lang("Please type in the absolute path to an existing terminal emulator."),
```

Comment:
Maybe I shouldn't interfere at this time, but I think replacing "type in" with "enter" would make sense.